serverlessworkflow/modules/ROOT/pages/persistence/persistence-with-postgresql.adoc

```diff
@@ -1 +1,166 @@
+
+In Kogito, persistence is enabled through addons. Although multiple addons are available for persistence, this guide focus on using the `kogito-addons-quarkus-persistence-jdbc` addon based on JDBC along with PostgreSQL. This addon, also extends on the Quarkus capabilities for JDBC and you can also use any of the features available directly from Quarkus JDBC support. For more information on Quarkus and JDBC, visit {quarkus_datasource_guide}[Quarkus Datasources].
+
+[[postgresql-persistence-example]]
+== Solution
+It is recommended that you follow the instructions in the next sections step by step. However, you check a completed example below with all steps applied.
+
+.Clone `kogito-examples` repository
+[source,shell]
+----
+git clone git@github.com:kiegroup/kogito-examples.git
+----
+
+The solution is located in the `serverless-workflow-examples/serverless-workflow-callback-quarkus` directory.
+
+Follow the `README` file instructions contained in the root of the project in order to execute it.
+
+[[postgresql-persistence-maven-dependencies]]
+== Add dependencies to Maven project
+
+To use the persistence add-on, add the following code to the `pom.xml` file of your project:
+
+.JDBC persistence add-on
+[source,xml]
+----
+<dependency>
+  <groupId>org.kie.kogito</groupId>
+  <artifactId>kogito-addons-quarkus-persistence-jdbc</artifactId>
+</dependency>
+----
+
+.Quarkus JDBC PostgreSQL
+[source,xml]
+----
+<dependency>
+  <groupId>io.quarkus</groupId>
+  <artifactId>quarkus-jdbc-postgresql</artifactId>
+</dependency>
+----
+
+.Quarkus Agroal Data Source
+[source,xml]
+----
+<dependency>
+  <groupId>io.quarkus</groupId>
+  <artifactId>quarkus-agroal</artifactId>
+</dependency>
+----
+
+[[postgresql-persistence-application-properties]]
+== Configure application.properties
+
+The following properties should be added to your project's application.properties file:
+
+.application.properties
+[source, properties]
+----
+kogito.persistence.type=jdbc
+----
+
+The following Quarkus properties are also recommended to be added:
+
+.application.properties
+[source, properties]
+----
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.username=postgres
+quarkus.datasource.password=pass
+quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/postgres
+----
+
+[[postgresql-persistence-database-schema]]
+== Create PostgreSQL database schema
+
+By default, the persistence addon creates the required database schema structure. That is usually only recommended for easy of development and can be turned off by setting `kogito.persistence.auto.ddl=false`.
+DDL scripts are available in case there is need to manually create the required database schema. These scripts are included in the https://repository.jboss.org/org/kie/kogito/kogito-ddl/{version}/kogito-ddl-{version}-db-scripts.zip[kogito-ddl-{version}-db-scripts.zip] artifact.
```

Comment:
```suggestion
DDL scripts are available in case there is a need to manually create the required database schema. These scripts are included in the https://repository.jboss.org/org/kie/kogito/kogito-ddl/{version}/kogito-ddl-{version}-db-scripts.zip[kogito-ddl-{version}-db-scripts.zip] artifact.
```